CMakeLists.txt

```diff
@@ -233,6 +235,22 @@ else()
 				set(CMAKE_C_FLAGS "-O0 -g")
 			endif()
 		endif()
+		string(TOUPPER "${WANT_SANITIZER}" WANT_SANITIZER )
+		if(WANT_SANITIZER STREQUAL ADDRESS)
+			set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=undefined -fsanitize=address -fno-omit-frame-pointer")
+			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=undefined -fsanitize=address -fno-omit-frame-pointer")
+			set(CMAKE_STATUS_DEBUG_SUPPORT "Yes, with address,leak and undefined behavior sanitizers")
+		elseif(WANT_SANITIZER STREQUAL MEMORY)
+			set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=memory -fPIE -pie -fno-omit-frame-pointer")
+			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=memory -fPIE -pie -fno-omit-frame-pointer")
+			set(CMAKE_STATUS_DEBUG_SUPPORT "Yes, with memory sanitizer")
+		elseif(WANT_SANITIZER STREQUAL THREAD)
+			set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=thread -fno-omit-frame-pointer")
+			set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fsanitize=thread -fno-omit-frame-pointer")
+			set(CMAKE_STATUS_DEBUG_SUPPORT "Yes, with memory sanitizer")
```

Comment:
that should probably be thread

Reply:
Fixed in https://github.com/kvirc/KVIrc/pull/2599/commits/42009c2d1eefaebf63b5d860b3cc3366537ee838, thank you